README.md

```diff
@@ -15,27 +15,21 @@ for a number of services including [the Snap Store](https://snapcraft.io/store),
 the majority of Canonical's web properties including [ubuntu.com](https://ubuntu.com) and
 [canonical.com](https://canonical.com), and [Ubuntu Extended Security Maintenance](https://ubuntu.com/security/esm).
 
-This Kubernetes-based version is built using the same approach, and can be
-used as a front-end caching service in a situation where your Kubernetes
-cluster and its ingress controllers have a fast connection to the Internet.
+This Kubernetes-based version is built using the same approach as the [machine content-cache charm](https://charmhub.io/content-cache), and can be used as a front-end caching service in
+a situation where your Kubernetes cluster and its ingress controllers have
+a fast connection to the Internet.
 
-## Usage
+## Project and community
 
-For details on using Kubernetes with Juju [see here](https://juju.is/docs/kubernetes), and for
-details on using Juju with MicroK8s for easy local testing [see here](https://juju.is/docs/microk8s-cloud).
-
-To deploy this charm into a k8s model:
-
-    juju deploy content-cache-k8s --channel edge
-    juju deploy hello-kubecon
-    juju deploy nginx-ingress-integrator
-    juju relate hello-kubecon content-cache-k8s:ingress-proxy
-    juju relate nginx-ingress-integrator content-cache-k8s:ingress
-
-You can then reach the site at `http://hello-kubecon` assuming `hello-kubecon`
-resolves to the IP address of your Kubernetes cluster (if you're on MicroK8s
-this will be 127.0.0.1).
+The Content-cache-k8s Operator is a member of the Ubuntu family. It's an
+open source project that warmly welcomes community projects, contributions,
+suggestions, fixes and constructive feedback.
+* [Code of conduct](https://ubuntu.com/community/code-of-conduct)
+* [Get support](https://discourse.charmhub.io/)
+* [Join our online chat](https://chat.charmhub.io/charmhub/channels/charm-dev)
+* [Contribute](https://charmhub.io/content-cache-k8s/docs/contributing)
```

Comment:
This link doesn't seem to be right - it's actually https://charmhub.io/content-cache-k8s/docs/how-to-guides-contributing